EIPS/eip-eof-metadata.md

```diff
@@ -0,0 +1,88 @@
+---
+eip: xxx
```

Comment:
```suggestion
eip: 7834
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.